usaspending_api/references/v2/views/autocomplete.py

```diff
@@ -204,23 +204,44 @@ def post(self, request):
 class RecipientAutocompleteViewSet(BaseAutocompleteViewSet):
 
     def post(self, request):
-        """Return all recipients matching the provided search text"""
+        """Return all Parents and Recipients matching the provided search text"""
 
         search_text, limit = self.get_request_payload(request)
 
-        queryset = LegalEntity.objects.all()
+        # Return Recipients with valid id entries
+        queryset = LegalEntity.objects.exclude(
+            recipient_unique_id__isnull=True).exclude(
+            recipient_unique_id__exact='')
 
-        queryset = queryset.annotate(similarity=Greatest(TrigramSimilarity('recipient_name', search_text),
-                                                         TrigramSimilarity('recipient_unique_id', search_text))).\
-            order_by('-similarity')
-
-        exact_match_queryset = queryset.filter(similarity=1.0)
-        if exact_match_queryset.count() > 0:
-            queryset = exact_match_queryset
-
-        column_names = ['legal_entity_id', 'recipient_name', 'recipient_unique_id']
-
-        return Response({'results': list(queryset.values(*column_names)[:limit])})
+        # Filter based on search text
+        response = {}
+        # Search and filter for Parent Recipients
+        try:
```

Comment:
Since there really aren't any exceptions to be raised, it's passed. So yes, it would just be empty. I just updated the gist to show what it would look like for a search on a recipient DUNS.